tests/Doctrine/Tests/Common/Persistence/ManagerRegistryTest.php

```diff
@@ -64,12 +63,12 @@ public function testGetManagerForInvalidAliasedClass()
     }
 }
 
-class TestManager
+class TestManager extends DoctrineTestCase
```

Comment:
Do we need to extend the `DoctrineTestCase`? Can't we just extend a `PHPUnit_Framework_TestCase`?